components/post_markdown/system_message_helpers.jsx

```diff
@@ -72,6 +84,22 @@ function renderAddToChannelMessage(post) {
     );
 }
 
+function renderAddGuestToChannelMessage(post) {
+    const username = renderUsername(post.props.username);
+    const addedUsername = renderUsername(post.props.addedUsername);
+
+    return (
+        <FormattedMessage
+            id='api.channel.add_guest.added'
+            defaultMessage='{addedUsername} added to the channel as guest by {username}.'
+            values={{
```

Comment:
Please update to : "{addedUsername} added to the channel as a guest by {username}."